app/src/main/java/com/materiapps/gloom/ui/screens/profile/ProfileScreen.kt

```diff
@@ -161,16 +158,16 @@ class ProfileScreen(
 
     @Composable
     private fun Header(
-        user: ProfileQuery.Viewer
+        user: ModelUser
     ) {
         Column(
             horizontalAlignment = Alignment.CenterHorizontally,
             verticalArrangement = Arrangement.spacedBy(5.dp),
             modifier = Modifier.fillMaxWidth()
         ) {
             AsyncImage(
-                model = user.avatarUrl,
-                contentDescription = "${user.name}'s avatar",
+                model = user.avatar,
```

Comment:
Is the avatar for deleted users still the ghost image or null

Reply:
idk, haven't tested